sql/policy_internal.sql

```diff
@@ -65,6 +65,11 @@ BEGIN
     lag := _timescaledb_internal.subtract_integer_from_now(htoid, lag::BIGINT);
   END IF;
 
+  -- chunk status bits:
+  -- 1: compressed
+  -- 2: compressed unordered
+  -- 4: frozen
+  -- 8: compressed partial
```

Comment:
We probably need to update the high-level description of our compression logic. Do we have one somewhere?

Reply:
Hmm i dont think we have what you are asking here atm.